### DISCUSSION TITLE
Is there a way to show more than 1 item in the array?

### DISCUSSION THREAD

**Discussion by Revester** [ReClassNET/ReClass.NET#189] [Q&A] [answered]
I have a raw array of floats of size 100: `float flBuffer[100]`. It's sitting in the middle of the class. If I mark it as an array, I can then view every element of the array only one at a time:
![ReClass NET_2Wo2W9xOha](https://user-images.githubusercontent.com/59390647/106469566-1f755a80-64a8-11eb-9c8f-655b725fbf98.png)

Or I can mark all 100 floats just as float and view them in a class, then these floats bloat the class. I'd like to be able to have the view that array provides: minimize and maximize feature but also to be able to view more than 1 item at a time. Maybe show like 30 of elements at a time with a scroller inside, in case there is a 1000 or 10000 elements in the array.

Is there some setting like that?

Also, when I'm viewing the array, I'd like to be able to determine which(indexes) elements are shown, maybe after the offset.

If I wanted to make this a reality, which classes I should look into?

**Comment by KN4CK3R** [comment, marked as answer]
Currently you can only view a single item of an array. If you want to implement that have a look at the `BaseWrapperArrayNode` class. There you can draw more items.

**Reply by Revester** [reply]
Thanks.
